client/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
whould have been good to add google font icons somewhere in the head, 
<link href="https://fonts.googleapis.com/icon?family=Material+Icons"
      rel="stylesheet">
This would let you infor icons